packages/app/script-utils/src/vault/vault.ts

```diff
@@ -100,7 +100,9 @@ export const vaultGetVars = (service: string): Record<string, string> => {
     return responseJson.data.data
   } catch (e: unknown) {
     if (isExecError(e)) {
-      globalLogger.error(`Vault ${service} error downloading env vars -> ${e.stderr.toString()}`)
+      globalLogger.error(
+        `Vault ${service} error downloading env vars -> ${e.stderr.toString()}. Make sure you're using correct path to secrets. Vault reports 403 if the path is incorrect.`,
```

Comment:
can't we do simple "file exists" check to throw an error if file does not exist?

Reply:
But this is response from vault cli, it reports 403 always if secrets path doesn't exist. Did you mean file exist check for `.env` file?

Reply:
I was confused about what kind of paths we talk about, all good :D